sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/session/SessionManager.java

```diff
@@ -148,6 +148,15 @@ public synchronized void start() {
     }
   }
 
+  private void initSessionTimeoutCheckerConfig() {
```

Comment:
should we refactor createBackgroundOperationPool to call this?

Reply:
Thanks @felixcheung . In my opinion you can create a new or follow-up PR if it is necessary. This PR is to fix the bug about the Session Timeout Checker does not work currently.

Reply:
I asked because these lines are duplicated
https://github.com/zuotingbing/spark/blob/34b7680dfd915449087b8f5541c682b02df5dfbc/sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/session/SessionManager.java#L105